src/AzureIoTHub.Portal/Server/Managers/DeviceProvisioningServiceManager.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace AzureIoTHub.Portal.Server.Managers
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Provisioning.Service;
+    using Microsoft.Azure.Devices.Shared;
+    using Microsoft.Extensions.Configuration;
+
+    public class DeviceProvisioningServiceManager : IDeviceProvisioningServiceManager
+    {
+        private readonly IConfiguration configuration;
+        private readonly ProvisioningServiceClient dps;
+
+        public DeviceProvisioningServiceManager(IConfiguration configuration, ProvisioningServiceClient dps)
```

Comment:
Instead of using ``IConfiguration``, please use ``ConfigHandler`` inside the manager.

You'll have to register the ConfigHandler inside the DI. You can do this at [https://github.com/CGI-FR/IoT-Hub-Portal/blob/39d2e00fa1494f77fbf1ee4557abbf140effc342/src/AzureIoTHub.Portal/Server/Startup.cs#L47](https://github.com/CGI-FR/IoT-Hub-Portal/blob/39d2e00fa1494f77fbf1ee4557abbf140effc342/src/AzureIoTHub.Portal/Server/Startup.cs#L47)
```
services.AddSingleton(configuration);
```